plugin/storage/cassandra/dependencystore/storage.go

```diff
@@ -46,7 +46,7 @@ func NewDependencyStore(
 ) *DependencyStore {
 	return &DependencyStore{
 		session:                  session,
-		dependenciesTableMetrics: casMetrics.NewTable(metricsFactory, "dependencies"),
+		dependenciesTableMetrics: casMetrics.NewTable(metricsFactory.Namespace("write", nil), "dependencies"),
```

Comment:
dependency store does both reads and writes, why do we want to namespace the whole factory?